### ISSUE TITLE
More and verbose income statistics

### ISSUE THREAD

**Issue by d-albrecht** [Sesu8642/FeudalTactics#22]
Could we get a more detailed income statistics? Just Slay apparently has. With additional/separate values for (a) income based on territories, (b) salary for units, (c) savings from last round.

On that note, could you change the income statistics to NOT having an effect when I pick up a unit? Say I have a kingdom of four tiles with a peasant and one tree. If I just select the kingdom, the income says "1", if I pick up the peasant the income says "3" which is quite confusing. Because this number is never correct and just keeping the "1" would in most cases closer to the actual truth. Especially in phases where I wait for money to accumulate, I often get a wrong impression of the number of turns this will take, when I happen to pick up the unit instead of just selecting the kingdom.

**Comment by Sesu8642** [comment]
I don't want to use too much screen space for the savings. Maybe if there was another place to display this more detailed information.

Picking up units will no longer have an effect on the income value. It is fixed in master and will be in the next release.

**Comment by d-albrecht** [comment]
I mean you could add those numbers in a very concise way. Currently you have "amount of money (expected change)" like in "42 (+23)" in one line. You can just add two more numbers within the "change" part like in "42 (+50 -27 = +23)". You can't tell me that this would take up too much space, or can you? But in the end, it's just a suggestion. So, no pressure.

It's just that even if I memorized all the unit costs (which I still didn't), I couldn't estimate a kingdom's size easily because this information isn't shown anywhere and I'm not counting tiles each round. If I had this number I knew for sure, for example "a knight isn't sustainable at this stage" without having to make one.

**Comment by Sesu8642** [comment]
Yes, a concise version like that would probably fit. It doesn't contain the savings from the last round but I personally don't see a lot of value in knowing that in the first place. And it can be calculated easily from the other values at the beginning of the turn.

**Comment by d-albrecht** [comment]
I mentioned the savings because Slay notes it down like this. But if I got that right in the videos I watched, Slay doesn't even predict the changes to the next turn, but only the changes since last turn. So, every unit you buy will have an effect on your next turn's balance, but the game doesn't tell you that. Your game is already more user friendly in this aspect. Hence, the current balance is more important than last turn's savings. Or these two might just be the same, depending on your interpretation. ;)
